### ISSUE TITLE
Should we be making AIRR releases citable?

### ISSUE THREAD

**Issue by bcorrie** [airr-community/airr-standards#467]
There is a mechanism for this linking Github and Zenodo.

https://guides.github.com/activities/citable-code/

**Comment by bcorrie** [comment]
Do we want to consider this? It looks like the AIRR Community has a Zenodo record and @bussec manages this.

https://zenodo.org/communities/airr-community/records?q=&l=list&p=1&s=10&sort=newest

The process one follows is enables github hooks to Zenodo, then when you tag a release it is automatically published to Zenodo.

We do this for iReceptor releases (e.g. https://zenodo.org/records/7430516)

**Comment by javh** [comment]
Seems like an extra layer to me, as we'll have a v2 paper to cite.

**Comment by bcorrie** [comment]
Not sure I agree - this isn't a question of whether we should make the v2.0 release citable, it is whether we should make all versions citable. We will (we hope) have a paper that matches the v2.0 release, but there is no real mechanism to properly cite any of the intermediate releases. From a FAIR software (or even FAIR standards) and reproducible research perspective, how can I cite that I used v1.4.1 of the python library. Or that my data is in a format that adheres to v1.4.1 AIRR specification (not 1.0, not 1.2, not 2.0).

Yes, you can point to PyPi or a github release tag, but neither are the equivalent of having a DOI/PID for that release.

**Comment by javh** [comment]
What would be the benefit of citing a specific version as opposed to just stating the version when you cite the paper? Would this be instead of citing the paper?

**Comment by bussec** [comment]
For the record: We have been doing this for the last 6 years ;-) see here [DOI:10.5281/zenodo.1185414](https://zenodo.org/doi/10.5281/zenodo.1185414). Somehow the automatic process broke between 1.3.1 and 1.4, I will fix this with the next days.

**Comment by javh** [comment]
lol. Well, that settles that then. :)

**Comment by bcorrie** [comment]
> For the record: We have been doing this for the last 6 years ;-) see here [DOI:10.5281/zenodo.1185414](https://zenodo.org/doi/10.5281/zenodo.1185414). Somehow the automatic process broke between 1.3.1 and 1.4, I will fix this with the next days.

Excellent...

**Comment by bussec** [comment]
I now had a closer look at this, but I could not figure out *why* the v1.4.1 and v1.5.0 releases did not trigger the archiving routine, potentially the token expired or was accidentally deleted. I then created a new webhook for the AIRR Standards repo which has been successfully tested and I used it to manually push the v1.4.1 and v1.5.0 releases to Zenodo (using https://github.com/zenodo/zenodo/issues/1463 as a cheat sheet). So I think this issue is solved now, we should however confirm that the next release does trigger the archiving again.
